src/style.css

```
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir-Book", sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy", sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
```

```suggestion
@font-face {
  font-family: "Avenir";
  src: url(./fonts/Avenir-Book.ttf);
  font-weight: 300;
}

@font-face {
  font-family: "Avenir-Heavy";
  src: url(./fonts/Avenir-Heavy.ttf);
  font-weight: 700;
}
```
You need to use only one font-family but different font-weights.
(also, 'bold' is 700 (not 800))